pkg/cluster/spec/validate.go

```diff
@@ -794,6 +794,22 @@ func (s *Specification) validateTLSEnabled() error {
 	return nil
 }
 
+func (s *Specification) validatePdPodNames() error {
```

Comment:
prefer to use the name `validatePDNames` because `Pod` is a term of K8S and we can really deploy TiDB to K8S. So we'd better change it to avoid misunderstanding